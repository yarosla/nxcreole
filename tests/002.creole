This is a test
  file.

 Это
проверочный
файл.
