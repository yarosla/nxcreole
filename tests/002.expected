<p>This is a test
file.</p>
<p>Это
проверочный
файл.</p>
